.github/workflows/heartbeat.yml

```diff
@@ -33,10 +33,10 @@ jobs:
     - name: Initiate Ritual
       run: .github/scripts/initiate_dkg.sh
       env:
-
         # Secret environment variables (secrets)
         APE_ACCOUNTS_automation_PASSPHRASE: ${{ secrets.DKG_INITIATOR_PASSPHRASE }}
         WEB3_INFURA_API_KEY: ${{ secrets.WEB3_INFURA_API_KEY }}
+        WEB3_INFURA_API_SECRET: ${{ secrets.WEB3_INFURA_API_SECRET }}
```

Comment:
Do we need both of these?

Reply:
> Do we need both of these - API_SECRET and API_KEY ?

Yes. The API_KEY is the Infura API Key as we already know and the API_SECRET is a "password" that brings an additional security measure. By this way, when ape-etherscan plughin uses both, it is possible to override the allowlist on our `Github Actions` Infura API key:

![image](https://github.com/user-attachments/assets/eb293c4d-6a83-4f1c-a1e3-3b12df934e6e)

Reply:
This is the `Github Actions` API Key? Did you add a API_SECRET or was it there before? If so, will that cause actions from other repos to fail?